wallet/src/Pos/Util/Mnemonics.hs

```diff
@@ -67,6 +69,8 @@ toMnemonic ent = do
 -- mnemonic.
 fromMnemonic :: Mnemonic -> Either String Entropy
 fromMnemonic ms = do
+    when (word_count <= 0) $
```

Comment:
Would `word_count <= 12` be better? As far as I know, in every context we require at least 12.

Reply:
We currently accept entropy between 4 and 64 bytes (i.e. 3 and 48 words). The BIP-39 official spec mentions the following:

> The mnemonic must encode entropy in a multiple of 32 bits. With more entropy security is improved but the sentence length increases. We refer to the initial entropy length as ENT. The allowed size of ENT is 128-256 bits. 

So, between 16 and 32 bytes (i.e. 12 and 24 words). I'd rather stick with BIP-39, however, I am afraid of the consequences of changing those boundaries at this stage :/